.github/workflows/metal_plugin_ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false # don't cancel all jobs on failure
       matrix:
-        jaxlib-version: ["plugin_latest"]
+        jaxlib-version: ["plugin_latest", "nightly"]
```

Comment:
nit: I would rename "plugin_latest" to "pypi_latest". The plugin is the same version in either case (latest), you're really just changing the jaxlib version, so it's confusing to mention "plugin" in the jaxlib version config.